service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java

```diff
@@ -139,29 +141,55 @@ void aboutToSubmitShouldPopulateListServedCaseManagementOrdersWhenSendsToAllPart
     }
 
     @Test
-    void aboutToSubmitShouldSendNotificationWhenStatusIsSendToJudge() throws Exception {
+    void aboutToSubmitShouldNotifyHmctsAdminWhenStatusIsSendToJudgeAndCtscIsDisabled() throws Exception {
         CaseManagementOrder order = buildOrder(SEND_TO_JUDGE, SEND_TO_ALL_PARTIES);
 
-        CaseDetails caseDetails = buildCaseDetails(order, DRAFT_CASE_MANAGEMENT_ORDER);
+        CaseDetails caseDetails = buildCaseDetails(order, DRAFT_CASE_MANAGEMENT_ORDER, "No");
 
         postAboutToSubmitEvent(buildCallbackRequest(caseDetails, DRAFT_CASE_MANAGEMENT_ORDER));
 
         verify(notificationClient).sendEmail(
-            eq(CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE), eq("admin@family-court.com"),
+            eq(CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE), eq(HMCTS_ADMIN_INBOX),
+            eq(expectedCMODraftCompleteNotificationParameters()), eq(caseId.toString()));
```

Comment:
I don't know about you, but I prefer each eq to be on a new line

Reply:
Sure